dlpy/tests/test_model.py

```diff
@@ -720,6 +721,43 @@ def test_model28(self):
         model1 = Model.from_onnx_model(self.s, m, offsets=[1, 1, 1,], scale=2, std='std')
         model1.print_summary()
 
+    def test_evaluate_obj_det(self):
+
+        caslib, path = caslibify(self.s, path = self.data_dir + 'evaluate_obj_det_det.sashdat', task = 'load')
+
+        if self.data_dir is None:
```

Comment:
can you please take this line above. so that if there is no data_dir defined it does not try to caslibify it